gax-grpc/src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -72,6 +72,11 @@
     return ((OperationsStubSettings) getStubSettings()).deleteOperationSettings();
   }
 
+  /** Returns the object with the settings used for calls to waitOperation. */
+  public UnaryCallSettings<WaitOperationRequest, Operation> waitOperationSettings() {
```

Comment:
this looks testable

Reply:
This is a direct import of the generated GAPIC surface for the Wait* portion of the LRO proto, so none of the `*Settings()` getters in this file are tested (as is the case in all the generated GAPIC libraries).

Reply:
I'm trying to understand whether you're proposing re-generating code in the future. That's not longterm maintainable. There are two paths that can work. Either the code is generated every time when the project is compiled, with the generated code never checked into the repo (the protobuf approach) or it's generated once, checked into the repo, and then manually edited for ever after.  Anything in between makes maintenance extremely difficult. Which path is this PR proposing?

Reply:
Ping. This is a critical issue.

Reply:
This PR (and others in this repo) only updates the LRO client with the auto-generated section for Wait, and does not propose a particular path. While the long-term path is not clear yet, keeping this client as consistent as possible with the generated code will make debugging less error prone and make updates easier, while we figure this out.

Reply:
Autogenerated or no., I still think a test helps here. If the autogenerated code chages in a way that causes the test to fail, the test will make that apparent.